bin/prettier.js

```diff
@@ -351,12 +358,20 @@ if (stdin) {
         process.exitCode = 2;
       }
     } else if (!argv["list-different"]) {
-      // Don't use `console.log` here since it adds an extra newline at the end.
-      process.stdout.write(output);
+      writeOutput(result);
     }
   });
 }
 
+function writeOutput(result) {
+  if (options.cursorOffset) {
+    console.log(result.cursorOffset);
```

Comment:
Do we just output the cursor offset from CLI? We don't output the formatted text? Could we print the formatted text on stdout and the cursor on stderr?